OSDOCS:13913 Add reason for no Node Disrupt Policy for /etc/containers/registries.conf.d (#91476)
openshift/openshift-docs: modules/machine-config-node-disruption-example.adoc

  observedGeneration: 9
----

> The default node disruption policy does not contain a policy for changes to the `/etc/containers/registries.conf.d` file. This is because both {product-title} and {op-system-base-full} use the `registries.conf.d` file to specify aliases for image short names. It is recommended that you always pull an image by its fully-qualified name. This is particularly true with public registries, as the image might not deploy if the public registry requires authentication. You can create a user-defined policy to use with the the `/etc/containers/registries.conf.d` file, if needed. However, you should be aware of this recommendation. 

Reviewer: Double "the"

Per SSG, should avoid "recommend" if possible, in favor of just telling them what they should do. Could we just shorten to tell them to always pull? Made some suggestions below, though I'm not sure what to do with that last sentence mentioning the recommendation again. Maybe it's either not necessary now, or could be reworded

as -> because, per ISG

```suggestion
The default node disruption policy does not contain a policy for changes to the `/etc/containers/registries.conf.d` file. This is because both {product-title} and {op-system-base-full} use the `registries.conf.d` file to specify aliases for image short names. Always pull an image by its fully-qualified name. This is particularly important with public registries, because the image might not deploy if the public registry requires authentication. You can create a user-defined policy to use with the `/etc/containers/registries.conf.d` file, if needed. However, you should be aware of this recommendation. 
```

Author: > Could we just shorten to tell them to always pull?

No. The ["short names" feature was specifically added](https://issues.redhat.com/browse/OCPNODE-371) to allow pull using unqualified domain names. Maybe I will use the [wording in the "short names" doc](https://github.com/openshift/openshift-docs/pull/28152/files#diff-6fb65b5ed05a3f67d1cee2098cc948d07c5d8d31c6aca5b234b350bd21dc48caR17-R22).  I wanted a more positive spin here.

Author: Loves that change: 
> This is particularly important with public registries

I think I will remove the last sentence. (TBH, I didn't like it either and was fishing for better wording!)

Author: I think I will remove the last sentence. (TBH, I didn't like it either and was fishing for better wording!)
